client.lua

```diff
@@ -5,19 +5,36 @@ local pointCamCoords = 75
 local pointCamCoords2 = 0
 local cam1Time = 500
 local cam2Time = 1000
-local choosingSpawn = false
+local isChoosingSpawn = false
 local Houses = {}
 local cam = nil
 local cam2 = nil
 
 -- Functions
 
-local function SetDisplay(bool)
-    choosingSpawn = bool
-    SetNuiFocus(bool, bool)
+-- Stops player from moving while choosing spawn
+local function launchDisableControlsThread()
+    CreateThread(function()
+        while isChoosingSpawn do
+            Wait(0)
+            DisableAllControlActions(0)
+        end
+    end)
+end
+
+--- Displays the spawn UI and disables controls
+---@param isChoosingSpawn boolean
+local function SetDisplay(isChoosingSpawn)
```

Comment:
param named the same as variable in file space?

Reply:
fixed in next commit